protocols/union/overview.md

```diff
@@ -0,0 +1,21 @@
+# Union
+
+## Overview
```

Comment:
New lines for all the subheaders in this file too.

Reply:
Demo of markdown formatting https://hackmd.io/JSKzs_V0R--TP7bmHstn0w